.packit.yaml

```diff
@@ -20,19 +14,45 @@ upstream_package_name: coreos-installer
 # downstream (Fedora) RPM package name
 downstream_package_name: rust-coreos-installer
 
+packages:
+  coreos-installer-fedora:
+    specfile_path: rust-coreos-installer.spec
+    actions:
+      post-upstream-clone:
+        - wget https://src.fedoraproject.org/rpms/rust-coreos-installer/raw/rawhide/f/rust-coreos-installer.spec
+      changelog-entry:
+        - bash -c 'echo "- New upstream release"'
+  coreos-installer-centos:
+    pkg_tool: centpkg
+    specfile_path: rust-coreos-installer.spec
+    actions:
+      post-upstream-clone:
+        - wget https://gitlab.com/redhat/centos-stream/rpms/rust-coreos-installer/-/raw/c9s/rust-coreos-installer.spec
+      changelog-entry:
+        - bash -c 'echo "- New upstream release"'
+
 jobs: 
 
 - job: propose_downstream
   trigger: release
+  packages: [coreos-installer-fedora]
   dist_git_branches:
     - fedora-all
 
 - job: koji_build
   trigger: commit
+  packages: [coreos-installer-fedora]
   dist_git_branches:
     - fedora-all
 
 - job: bodhi_update
   trigger: commit
+  packages: [coreos-installer-fedora]
   dist_git_branches:
     - fedora-all
+
+- job: propose_downstream
+  trigger: release
+  packages: [coreos-installer-centos]
+  dist_git_branches:
+    - c9s
```

Comment:
Might as well add c10s now. Or is Packit integration not yet done for that yet?

Reply:
err, at least according to docs https://packit.dev/docs/configuration/upstream/propose_downstream#syncing-the-release-to-centos-stream c8s and c9s are it.

Reply:
Does this mean it will create PR and merge automatically, is it necessary to add comment like `resolve #<bug>` ?